curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804eb.md

```diff
@@ -9,7 +9,9 @@ dashedName: step-65
 
 The `title` element determines what browsers show in the title bar or tab for the page.
 
-Add a `title` element within the `head` element. Its text should be `CatPhotoApp`.
+Add a `title` element within the `head` element. Its text should be
```

Comment:
Maybe this would work better as one sentence. 

Add a `title` element within the `head` element using the following text below:

Reply:
Nice catches. 👍